osbuild/formats/v2.py

```diff
@@ -391,9 +391,22 @@ def load(description: Dict, index: Index) -> Manifest:
 
 
 #pylint: disable=too-many-branches
-def output(manifest: Manifest, res: Dict) -> Dict:
+def output(manifest: Manifest, res: Dict, store=None) -> Dict:
```

Comment:
(Github PR review will likely show this comment in an odd position. This is on `fmt/v2: extract metadata...`)

You add `store=None` as parameter but don't use it. You only end up using it later on. I don't mind it, but if you rebase this anyway, feel free to fix this.